node/bindnode/fuzz_test.go

```diff
@@ -132,6 +145,7 @@ func marshalDagJSON(tb testing.TB, node datamodel.Node) []byte {
 
 func FuzzBindnodeViaDagCBOR(f *testing.F) {
 	for _, input := range fuzzInputs {
+		// f.Logf("debug: %#v\n", input)
```

Comment:
remove fully?

Reply:
It's been useful to me twice already, so I thought it was harmless to leave behind. I don't feel strongly either way, as this is test code anyway. But keeping it uncommented always is unfortunate, because fuzzing begins by spamming a bunch of generally unhelpful stuff.

Reply:
I don't think we have a strong convention here. probably fine to leave as is.

Reply:
season to taste indeed, but I certainly don't object to leaving printfs like this behind in comments.

It's not always obvious what a useful printf incantation is going to be, so hints are cool imo.